dashboard/src/components/ChartView/ChartHeader.v2.tsx

```diff
@@ -0,0 +1,92 @@
+import Column from "components/js/Column";
```

Comment:
I think this is only looks untested because you actually just extracted it from the chartview below, so still covered by the tests there (I assume).

Reply:
yes, it's both tested with ChartView and the DeploymentForm